AzureData/Source/AzureData.swift

```diff
@@ -104,9 +104,13 @@ public func get (collectionWithId collectionId: String, inDatabase databaseId: S
 }
 
 // delete
-//public func delete (_ collection: DocumentCollection, fromDatabase databaseId: String, callback: @escaping (DataResponse) -> ()) {
-//    return DocumentClient.default.delete (collection, fromDatabase: databaseId, callback: callback)
-//}
+public func delete (_ collection: DocumentCollection, fromDatabase databaseId: String, callback: @escaping (DataResponse) -> ()) {
```

Comment:
Same here, if we have the `collection` I don't think we should need to pass in the `databaseId`.

Reply:
I agree. They have this override on Android and thought to add it for parity.